### DISCUSSION TITLE
cv2.error: OpenCV(4.5.5) :-1: error: (-5:Bad argument) in function 'resize'

### DISCUSSION THREAD

**Discussion by Mostafizur15** [datasciencecompleteguide/Machine-Learning#46] [General]
def rescaleFrame(frame,scale=0.75):
  width =  int(frame.shape[1]*scale)
  height = int(frame.shape[0]*scale)
  dimensions = (height*width)
  return cv.resize(frame, dimensions,interpolation=cv.INTER_AREA)

captur = cv.VideoCapture("vdo.mp4")

while True:
    isTrue, frame=captur.read()
    frame_resized=rescaleFrame(frame)

    cv.imshow("ok",frame)
    cv.imshow("Resized vedio",frame_resized)

    if cv.waitKey(1) & 0xff == ord('d'):
      break
captur.release()
cv.destroyAllWindows()

please tell me why this code giving me 

.--->>>>>>>>>> cv2.error: OpenCV(4.5.5) :-1: error: (-5:Bad argument) in function 'resize' 

this error....

**Comment by datasciencecompleteguide** [comment]
just put a comma in the dimensions tuple
i.e.,
dimensions = (height, width)
